src/test/groovy/com/palantir/gradle/javadist/JavaDistributionPluginTests.groovy

```diff
@@ -61,6 +61,7 @@ class JavaDistributionPluginTests extends GradleTestSpec {
         // try all of the service commands
         exec('dist/service-name-0.1/service/bin/init.sh', 'start') ==~ /(?m)Running 'service-name'\.\.\.\s+Started \(\d+\)\n/
         file('dist/service-name-0.1/var/log/service-name-startup.log', projectDir).text.contains('Test started\n')
+        exec('dist/service-name-0.1/service/bin/init.sh', 'start') ==~ /(?m)Process is already running\n/
```

Comment:
Yeah I just ran the test on develop and it passed (so doesn't catch the bug), working on making `exec()` check the return code. I think there's an assumption throughout this suite that it returns 0 so I can bake in an assertion - you cool with that?